app/models/ownership.rb

```diff
@@ -3,17 +3,4 @@ class Ownership < ActiveRecord::Base
   belongs_to :user
 
   validates :user_id, uniqueness: { scope: :rubygem_id }
-
-  before_destroy :keep_last_owner
```

Comment:
I am deleting `ownership` records associated with user as well. Previously it was not possible for a gem to have no owner, now with user delete possible it will be (`orphaned` gems).

I am guessing this check was in place so that users can't remove all the gem owners with `gem owner` command. I have ensure that they still won't be able to do it [here](https://github.com/rubygems/rubygems.org/pull/1301/files#diff-76999f2b6a3a61482baf0e780a052acaR29).